.ci/build_dattobd.gitlab-ci.yml

```diff
@@ -0,0 +1,115 @@
+#
+# Build dattobd on RHEL9
+#
+build:dattobd:on:rhel9:
+  stage: build dattobd
+  tags:
+    - docker-throttled
+  needs: ["create:rhel9:vm"]
+  script:
+    - apt-get update -y
+    - apt-get install sshpass -y
+    - sshpass -p "$VM_PASSWORD" ssh -o StrictHostKeyChecking=no datto@$RHEL9_VM_IP '
+        sudo rm -rf dattobd;
+        git clone https://github.com/krzysztof-wasieczko/dattobd.git --recursive;
```

Comment:
why it's ur private repo, not the official one?

Reply:
Good question , probably it's leftover from my tests. I've changed to original one.